packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1056,6 +1056,13 @@ export default class Meeting extends StatelessWebexPlugin {
         });
       }
     });
+
+    this.locusInfo.on(LOCUSINFO.EVENTS.MEDIA_INACTIVITY, (mediaStatus) => {
+      this.reconnect();
+      if (mediaStatus) {
+        // mediaStatus
```

Comment:
What is this?

Reply:
in future if we want to indicate the user that you are not sending media as expected and call might reconnect or disconnect